### PR TITLE
Add p9 radio

### PR DESCRIPTION
**Pull request by cuhome**
add CUAV p9 radio

### CODE REVIEW

**Comment by Hwurzburg** [comment]
needs to be added to telemetry page https://ardupilot.org/plane/docs/common-telemetry-landingpage.html

File: common/source/docs/common-telemetry-landingpage.rst [hunk 1]
```diff
@@ -42,6 +42,7 @@ Long Range
     Blicube RLINK P900 <common-blicube-rlink>
     CRSF Telemetry <common-crsf-telemetry>
     CUAV P8 Radio <common-cuav-p8>
+    CUAV P9 Radio <common-cuav-p9
```
Comment (Hwurzburg) [inline common/source/docs/common-telemetry-landingpage.rst:45]: left off the >

**Comment by Hwurzburg** [comment]
the table needed a lot of work, I have pushed the changes rather than make you do them..will push the corrections in a moment..however, the link to the user manual is broken and needs fixing before merging...

**Comment by cuhome** [comment]
@Hwurzburg 
ok, thank you, the user manual link is not broken, I am uploading it.

**Comment by Hwurzburg** [comment]
@cuhome let me know when the link is active and I will merge this...thanks

**Comment by Hwurzburg** [comment]
removed link and merging

**Pull request merged**